M2/Macaulay2/packages/SuperLinearAlgebra.m2

```diff
@@ -0,0 +1,758 @@
+newPackage(
+    "SuperLinearAlgebra", 
+    Version => "0.1", 
+    Date => "29 January 2021", 
+    Authors => {
+        {Name => "Fereshteh Bahadorykhalily", 
+         Email => "f.bahadori.khalili@gmail.com", 
+         HomePage => "https://www.researchgate.net/profile/Fereshte_Bahadorykhalily"
+        }, 
+        {Name => "Fatemeh Tarashi Kashani", 
+         Email => "tarashikashanifatemeh@gmail.com", 
+         HomePage => "https://www.linkedin.com/in/fatemehtarashi/"
+        }
+    }, 
+    Headline => "berezinian and supertrace of a supermatrix", 
+    DebuggingMode => true
```

Comment:
The debugging mode should be set to false, as the users will not be able to debug your package.